app/views/admin/graphs/tables.html.haml

```diff
@@ -2,7 +2,9 @@
   .col-md-6
     %h1
       Bike Index records tables
-      %small
+  .col-md-6
```

Comment:
It would be great if the admin graphs index nav list was used here too. Probably using shared partial